pyta_template.html

```diff
@@ -14,7 +14,158 @@
     <link rel="stylesheet" href="$css$" $if(html5)$$else$$$type="text/css" $endif$/>
   $endfor$
 </head>
+
 <body>
+
+<ul id="nav">
+  <li class="dropdown">
+    <a href="index.html#Elif checker" class="dropbtn">Elif checker</a>
```

Comment:
You don't need the link here, you can just use `href="#"` (applies to all sections).